projects/go-lib/src/lib/components/go-select/go-select.component.scss

```diff
@@ -12,24 +12,33 @@
     font-size: .875rem;
     font-weight: $weight-regular;
     letter-spacing: .5pt;
+    line-height: 1.3;
+    padding: 0;
   }
 }
 
 .ng-select.ng-select-single .ng-select-container,
 .ng-select.ng-select-multiple .ng-select-container {
-  height: 2.25rem;
```

Comment:
Is there a reason for removing this height here? If we keep it we can avoid adding the `position: relative !important;` below I believe.

Reply:
@StevenUlmer yes. This is being removed because we should not be specifying an exact height for this. The size of the element should depend upon the size of the font, line-height, and padding. If we were to change any of those it would affect the height. That's kind of why this PR exists.

Unfortunately `ng-select` applies styles directly to this element when it handles opening/closing which is why that `!important` is necessary.